config/management/src/lib.rs

```diff
@@ -126,6 +155,11 @@ fn secure_storage(config: SecureBackend) -> Box<dyn Storage> {
     ))
 }
 
+/// These tests depends on running Vault, which can be done by using the provided docker run script
+/// in `docker/vault/run.sh`.
+/// Note: Some of these tests may fail if you run them too quickly one after another due to data
+/// sychronization issues within Vault. It would seem the only way to fix it would be to restart
+/// the Vault service between runs.
```

Comment:
Could we hack it to run each test sequentially and force some sleeps() between runs? It would be nice to be able to just run 'cargo xtest --ignore' in this crate and have everything work as expected. If this doesn't interest you, feel free to add a TODO for me 😄

Reply:
Actually as I ran it more and more, I found it occurred randomly... my suspicion is there is some weird state in vault that this triggers but our vault test doesn't. I really hope we can get an internal backend soon :).